src/main/webapp/resources/js/pages/analysis/components/AnalysisJsonPreview.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * @File component renders a JSON preview of output files.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Divider, List, Row, Typography } from "antd";
+import { getDataViaChunks } from "../../../apis/analysis/analysis";
+import { isAdmin } from "../../../contexts/AnalysisContext";
+import { OutputFileHeader } from "../../../components/OutputFiles/OutputFileHeader";
+import { grey4 } from "../../../styles/colors";
+import { JsonOutputWrapper } from "../../../components/OutputFiles/JsonOutputWrapper";
+import { JsonObjectOutputWrapper } from "../../../components/OutputFiles/JsonObjectOutputWrapper";
+
+import AutoSizer from "react-virtualized-auto-sizer";
+import { FixedSizeList as VList } from "react-window";
+
+const { Text } = Typography;
+const SCROLLABLE_DIV_HEIGHT = 300;
+
+export default function AnalysisJsonPreview({ output }) {
+  let savedText = "";
+  const [jsonData, setJsonData] = useState(null);
+  /*
+   * Get json file output data and set the jsonData local state to this data.
+   */
+  useEffect(() => {
+    getDataViaChunks({
+      submissionId: output.analysisSubmissionId,
+      fileId: output.id,
+      seek: 0,
+      chunk: output.fileSizeBytes
+    }).then(data => {
+      savedText = data.text;
+      let parsedJson = JSON.parse(savedText);
+      let jsonListData = [];
+
+      if (Array.isArray(parsedJson)) {
+        Object.keys(parsedJson).map((key, val) => {
+          if (parsedJson[val] !== undefined) {
```

Comment:
Same this about checking `undefined` **ALWAYS** use 
```javascript
typeof WHATEVER === "undefined"`
```

Reply:
Updated in 5af0b378509161b98e4718f05132265df8b225c3